packages/@aws-cdk/cicd/lib/pipeline-deploy-stack-action.ts

```diff
@@ -0,0 +1,84 @@
+
+import cfn = require('@aws-cdk/aws-cloudformation');
+import api = require('@aws-cdk/aws-codepipeline-api');
```

Comment:
I would probably just call this import `codepipeline` (`api` is weirdly generic, doesn't really tell me anything).